frank-doc-doclet/src/test/java/org/frankframework/doc/FrankDocGroupValue.java

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2024 WeAreFrank!
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package org.frankframework.doc;
+
+public enum FrankDocGroupValue {
+	@EnumLabel("Pipes")
+	PIPES,
+	@EnumLabel("Senders")
+	SENDERS,
+	@EnumLabel("Listeners")
+	LISTENERS,
+	@EnumLabel("Validators")
+	VALIDATORS,
+	@EnumLabel("Wrappers")
+	WRAPPERS,
+	@EnumLabel("TransactionalStorages")
```

Comment:
I'd prefer the singular notation, instead of the plural ones. Can you change that for Pipes until TransactionalStorages? Both in the value as the enum property name. Thanks!

Reply:
Don't you want to change ERROR_MESSAGE_FORMATTERS?

Reply:
Oh oops, you're right, that one too. Thanks 👍🏻

Reply:
I am doing that one too.

Reply:
Gedaan.